src/macros.rs

```diff
@@ -720,3 +720,53 @@ macro_rules! const_unreachable {
         loop {}
     }};
 }
+
+/// Asserts at compile time that `$condition` is true for `Self` or the given
+/// `$tyvar`s.
+macro_rules! ct_assert {
```

Comment:
Is there a way to name this (and phrase the doc comment) so that it's clear why it's different from `const_assert` (immediately above in this file)?

Reply:
Note that this name will appear in compiler errors, so we want it to be both externally intelligible and succinct.

I can definitely improve the documentation, but would welcome suggestions of alternative names.

Reply:
Oh, I suppose `static_assert` might be fitting.

Reply:
[Done](https://github.com/google/zerocopy/compare/b2cb6b7adeff72a5dfa2d9a273308117574a28ed..6cfde9151b0b74d4f51e6cdb14a1dbd1ef7be5ad)